CHANGELOG.md

```diff
@@ -1,5 +1,112 @@
+- [Android][fixed] Make TurboModuleRegistry in CatalystInstanceImpl.java volatile ([29b9972](https://github.com/facebook/react-native/commit/29b9972) by [@RSNara](https://github.com/RSNara))
+- Refactor TurboModule filtering in NativeModuleRegistryBuilder ([d9deee2](https://github.com/facebook/react-native/commit/d9deee2) by [@RSNara](https://github.com/RSNara))
+- Support nullable returns NativeModule methods returning Boxed Primitives ([f57b0ca](https://github.com/facebook/react-native/commit/f57b0ca) by [@RSNara](https://github.com/RSNara))
+
+#### iOS specific
+
+- Remove \_\_unused annotation from ConcreteSystraceSection ctor args. ([8ac9b32](https://github.com/facebook/react-native/commit/8ac9b32) by [@RSNara](https://github.com/RSNara))
+- Slider is now disabled properly on iOS if the disabled prop is set. ([fa9ff07](https://github.com/facebook/react-native/commit/fa9ff07))
+- Fix iOS tests by using 12.4 iOS Simulator ([97dd8ce](https://github.com/facebook/react-native/commit/97dd8ce) by [@hramos](https://github.com/hramos))
+- Changelog: [iOS][fixed] Reconnect to debugger websocket after metro is restarted. ([13992f9](https://github.com/facebook/react-native/commit/13992f9) by [@rickhanlonii](https://github.com/rickhanlonii))
+- Fix spinner visibility on beginRefreshingProgrammatically ([e341489](https://github.com/facebook/react-native/commit/e341489) by [@nnabinh](https://github.com/nnabinh))
+- `Dimensions` module now updates on initial split screen ([e7c289b](https://github.com/facebook/react-native/commit/e7c289b) by [@sahrens](https://github.com/sahrens))
+
+### Unknown
+
+- Use vanilla jni instead of fbjni for all the jni communication ([f0f83eb](https://github.com/facebook/react-native/commit/f0f83eb) by [@SidharthGuglani](https://github.com/SidharthGuglani))
+- Introducing ReactCallerContextFactory interface ([9713b63](https://github.com/facebook/react-native/commit/9713b63) by [@mdvacca](https://github.com/mdvacca))
+- Integrate ReactContextContainerFactory as part of ReactImageViewManager ([882e4a3](https://github.com/facebook/react-native/commit/882e4a3) by [@mdvacca](https://github.com/mdvacca))
+- Deprecate CallerContext from ReactImageManager ([8accd77](https://github.com/facebook/react-native/commit/8accd77) by [@mdvacca](https://github.com/mdvacca))
+- Create ReactFeatureFlag to enable / disable custom implementation of getChildVisibleRect ([9c61693](https://github.com/facebook/react-native/commit/9c61693) by [@mdvacca](https://github.com/mdvacca))
+- Convert NetworkingModule to TurboModule ([476a336](https://github.com/facebook/react-native/commit/476a336) by [@ejanzer](https://github.com/ejanzer))
+- Avoid throwing exceptions when the host activity is not FragmentActivity ([7cfabf4](https://github.com/facebook/react-native/commit/7cfabf4))
+- Changelog: [General][adds] unstable_enableLogBox ([c194ed2](https://github.com/facebook/react-native/commit/c194ed2) by [@rickhanlonii](https://github.com/rickhanlonii))
```

Comment:
This will probably get put at the top of the changelog cc @rickhanlonii

Reply:
Probably we'll also reword this to said it's stable I imagine ;)